### DISCUSSION TITLE
DataGrid - Extensions

### DISCUSSION THREAD

**Discussion by marcosgerene** [irihitech/Ursa.Avalonia#128] [General]
I am looking for something that work as https://github.com/dotnet/DataGridExtensions works to WPF.

Is it somehow in plans? I downloaded the project and run the Demo, it was great, I just missed DataGrids, which are one of the biggest part of my UI/UX here.

**Comment by rabbitism** [comment]
I will try it, this solution is very promising.

**Comment by marcosgerene** [comment]
@rabbitism 

It has been a while since I came here the first time, I still have plans to migrate from WPF to Avalonia and Ursa seems to be a good choice.

I don't understand anything of Chinese, I translated that issue https://github.com/irihitech/Ursa.Avalonia/issues/88 and you commented something as "it is hard". Am I right?

My question: Is it (DataGrid) in your plans somehow?

**Reply by rabbitism** [reply]
Hi, it is in our plan, but we have limited developers and our priority is Date/Time picker atm. DataGrid will not happen in near future (at least this month)

**Reply by marcosgerene** [reply]
@rabbitism 

Thanks a lot on your time. I do understand your team-limitation, even more considering it is a open-source project. I am asking this because I notice there is not even a paid version of a DataGrid Advanced tool for Avalonia (I am talking about all vendors I know), interesting isn't it ?

Talking about feature, is there a roadmap for Ursa?

**Comment by liugangnhm** [comment]
i think this feature need to dig deeply into datagrid source code to implement. it is realy very hard.
